server/helpers.go

```diff
@@ -13,14 +12,18 @@ import (
 	"strconv"
 	"strings"
 
+	"github.com/StackExchange/wmi"
+
 	"github.com/JensvandeWiel/ArkAscendedServerManager/helpers"
 	"github.com/go-ini/ini"
 	"github.com/sethvargo/go-password/password"
 	"github.com/wailsapp/wails/v2/pkg/runtime"
 )
 
 var iniOpts = ini.LoadOptions{
-	AllowShadows: true,
+	// This setting allowed duplicate values in the INI files. With the Changes values should now be replaced
+	AllowShadows:            false,
```

Comment:
This can't be set to false, there are options such as OverridePlayerLevelEngramPoints that need to have multiple key value pairs.

Reply:
It has to be set as AllowShadows: true in the ini file

Reply:
If you want to be set to false It has to be set as AllowShadows: false in the ini like this similar to how AllowShadows: true is in the ini file

Reply:
Having AllowShadows as true and the Append change will help get settings correctly assigned on save. However for the purpose of having the ability for multiple key value pairs for OverridePlayerLevelEngramPoints, the ini Option AllowDuplicateShadowValues needs to be true. This however brings back the problem of duplicating values on every save of the config, which is a problem as well

Reply:
Yeah, its kind of unfixable in this way, so people should not use "use ini configuration" but they should just import their ini once and then change it through the app